src/coherence/client.py

```diff
@@ -1667,6 +1672,14 @@ def __init__(
         if tls_options is not None:
             self._tls_options = tls_options
 
+    async def resolve_ns_address(self) -> None:
```

Comment:
Missing doc if this is public.  If it's intended to be private/protected, it should start with '_'

Reply:
fixing this to be private/protected